just_audio/android/src/main/java/com/ryanheise/just_audio/AudioPlayer.java

```diff
@@ -660,10 +678,31 @@ public void setVolume(final float volume) {
     }
 
     public void setSpeed(final float speed) {
-        player.setPlaybackParameters(new PlaybackParameters(speed));
+        _speed = speed;
+        if (_skipSilence) {
+            player.setPlaybackParameters(new PlaybackParameters(_speed, 1, _skipSilence));
```

Comment:
Is there an issue with using this line unconditionally? (i.e. it seems to universally handle both the if/else cases)